save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/config/WebSecurityConfig.kt

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
can it be moved to common module and re-used (with customization) in backend and sandbox?

Reply:
Yes, but I plan to do it at the next step within #1299